incubator/nginx/examples/generated.yaml

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
It looks like we're generating the vhost in the code, but it doesn't show up here?

Reply:
My bad! Pushed wrong file.